src/components/organisms/AdminVenueView/components/MapBackgroundPlaceholder/MapBackgroundPlaceholder.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { CenterContent } from "components/atoms/CenterContent";
+
+import "./MapBackgroundPlaceholder.scss";
+
+export const MapBackgroundPlaceholder: React.FC = () => (
```

Comment:
Please move this component in `molecules`

Reply:
fixed in 2bb4a722ddb53f61dcec6f33062a584b8aa8c70c